engine/src/executionerrors.h

```diff
@@ -2778,8 +2778,16 @@ enum Exec_errors
     EE_BAD_PERMISSION_NAME,
     
     // {EE-0910} Property: value is not a data
-    EE_PROPERTY_NOTADATA
+    EE_PROPERTY_NOTADATA,
     
+    // {EE-0911} Math: domain error
+    EE_MATH_DOMAIN,
```

Comment:
We are probably safe to re-use the first DOMAIN / RANGE / DIV BY ZERO code that exists, and just mark all others as unused... So EE_ACOS_DOMAIN -> EE_MATH_DOMAIN (numeric: domain error (undefined)); EE_DIVIDE_RANGE -> EE_MATH_RANGE (numeric: range error (overflow)) ; EE_DIVIDE_ZERO -> EE_MATH_ZERO (numeric: divide by zero).

Reply:
Codes which they replace should be changed to EE_UNUSED_XXXX - where XXXX is the error code (formatted with 0s).